DXRVanilla/DeusEx/Classes/Player.uc

```diff
@@ -858,6 +858,16 @@ function bool IsFrobbable(actor A)
     return Super.IsFrobbable(A);
 }
 
+function CreateAimLaser()
+{
+    if (aimLaser == None) {
+        aimLaser = Spawn(class'DXRAimLaserEmitter', Self, , Location, Pawn(Owner).ViewRotation);
+            if (aimLaser == None) {
```

Comment:
awkward indentation here, at least as rendered in the Github website
```suggestion
        if (aimLaser == None) {
```

Reply:
Yeah, that was a result of not fixing indentation correctly when copying the code from elsewhere.